### DISCUSSION TITLE
missing link

### DISCUSSION THREAD

**Discussion by lolsenkettle** [Chaste/Chaste#317] [Q&A] [answered]
Hi,

Could you please direct me to this missing link which is currently broken? https://chaste.github.io/docs/user-tutorials/singlecellsimulation/

This line: https://chaste.cs.ox.ac.uk/trac/wiki/ChasteGuides/CodeGenerationFromCellML

Thanks, Louise

**Comment by mirams** [comment, marked as answer]
See https://chaste.github.io/docs/user-guides/code-generation-from-cellml/

**Comment by fcooper8472** [comment]
Thanks @lolsenkettle for spotting this!

I've opened [an issue](https://github.com/Chaste/Chaste.github.io/issues/102) on our website repository to correct this and other old links, which should be corrected shortly.
